code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -848,10 +848,10 @@
 		var/obj/vehicle/V = C
 		var/datum/component/riding/R = V.GetComponent(/datum/component/riding)
 		if(R)
-			if(R.vehicle_move_delay <= 0 )
+			if(R.vehicle_move_delay <= (CONFIG_GET(number/movedelay/run_delay) * num) )
```

Comment:
num is the number :) 

IE if you want to make the number larger by 20% you'd set num as 1.2